json-editor/json-editor.d.ts

```diff
@@ -39,7 +39,7 @@ type JSONEditorOptions<TValue> = {
     /**
      * The icon library to use for the editor.
      */
-    iconlib?: boolean;
+    iconlib?: string;
```

Comment:
Can you do the same thing here as we do for `show_errors`, since the list of supported options is finite.